source/js/components/join/language-select.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+export default class LanguageSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: "" };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+    console.log(`After: ${this.state.value}`);
```

Comment:
rogue console.log?

Reply:
lol yes...sorry, thanks for pointing it out.